### PR TITLE
Feature/api structure update

### PR DESCRIPTION
**Pull request by Foteno**
Įvertinkite, ačiū.

### CODE REVIEW

**Review by Tomas-Per — APPROVED** [review]

File: Backend/src/main/java/lt/vu/ads/controllers/OrderController.java [hunk 1]
```diff
@@ -1,104 +1,44 @@
...
-            throw new CustomException("Order time is empty ");
-        }
-        if(orderDetails.getConvenientArrivalTimeTo().before(orderDetails.getConvenientArrivalTimeFrom())) {
-            throw new CustomException("Order's time is not allowed");
-        }
-        order.setConvenientArrivalTimeFrom(orderDetails.getConvenientArrivalTimeFrom());
-        order.setConvenientArrivalTimeTo(orderDetails.getConvenientArrivalTimeTo());
-        final Order updatedOrder = orderRepository.save(order);
-        return ResponseEntity.ok(updatedOrder);
+    public ResponseEntity<Order> updateOrderd(@PathVariable(value = "id") Long orderId,
```
Comment (aule2000) [inline Backend/src/main/java/lt/vu/ads/controllers/OrderController.java]: Maybe there should be an updateOrder?

**Review by aule2000 — CHANGES_REQUESTED** [review]

File: Backend/src/main/java/lt/vu/ads/controllers/OrderController.java (hunk 1 above)
Comment (Foteno) [reply Backend/src/main/java/lt/vu/ads/controllers/OrderController.java]: oops

Comment (Foteno) [reply Backend/src/main/java/lt/vu/ads/controllers/OrderController.java]: I fixed it with 2nd commit, please check Files changed tab

**Review by Filari02 — APPROVED** [review]

File: Backend/src/main/java/lt/vu/ads/exceptions/ExceptionController.java [hunk 2]
```diff
@@ -20,21 +20,25 @@
 
 @ControllerAdvice
 @CommonsLog
-public class NotFoundExceptionController {
+public class ExceptionController {
     private static final String UNKNOWN_SERVER_ERROR_RESPONSE_MESSAGE = "Nežinoma serverio klaida";
```
Comment (aule2000) [inline Backend/src/main/java/lt/vu/ads/exceptions/ExceptionController.java]: Maybe you should write unknown server error message in english

**Review by aule2000 — CHANGES_REQUESTED** [review]

**Review by aule2000 — APPROVED** [review]

File: Backend/src/main/java/lt/vu/ads/controllers/OrderController.java [hunk 3]
```diff
@@ -1,104 +1,44 @@
...
-
-        if (source_address != null){
-            order.setSourceAddress(addressRepository.findOneById(source_address.getId()));
-        }
-        else{
-            addressController.createAddress(order.getSourceAddress());
-        }
-
-        return orderRepository.save(order);
+    public Order ecreateOrder(@RequestBody Order order) {
```
Comment (Tomas-Per) [inline Backend/src/main/java/lt/vu/ads/controllers/OrderController.java]: typo here probably

**Review by Ugniusst — APPROVED** [review]

**Pull request merged**